assignment7.txt

```diff
@@ -0,0 +1,19 @@
+Yael Kelmer. 
+
+1. ++*p increments the value of the datatype located at pointer p and then points at that location. *p++ points at the location in memory of p and then increments the value there.  
+
+2. The left to right order is guaranteed for operator precedence. 
+
+
+3. One advantage of pointers is that you can move along a string and point to different parts of it. 
```

Comment:
-Pointers are more efficient in handling arrays and data tables 
-They can be used to return multiple values from a function via function arguments 
-Pointers permit references to functions 
-Pointers allow C to support dynamic memory management 
-Pointers reduce length and complexity of programs